### ISSUE TITLE
Determining incident VTE

### ISSUE THREAD

**Issue by speed-vm** [opensafely/vte-diagnoses-and-deaths#15]
**Summary of meeting last week**

**Problem**
How to distinguish incident cases of VTE from prevalent cases? 
This is a problem because most patients tend to have 4 or 5 codes for VTE, so how do you know at the beginning of the study whether this is a new case.
A history of VTE increases likelihood of recurrent VTE, and don't want to exclude those at higher risk.

**Solution**
In our catch up @Jongmassey @joshuamutio @matthewberesford92 @speed-vm discussed important to define a period of time in which there should be no code for VTE in the period before the study and then during the study. 

@speed-vm  discussed with specialist team. Initial suggestion was 3 months but agreed too short as likely would coincide with an end of treatment review and the same with 6 months. Team have done a similar study but need to get in touch with first author to double check method. 

**Literature.** 
A Swedish study used **one year,** which seems reasonable (most VTE patients will stop treatment and follow up at 3 or 6 months).  [Link](https://www.tandfonline.com/doi/full/10.1080/03009742.2021.2001907)

They initially tried 90 days, but one year reduced the number of false positives.

The next thing to consider evidence of anticoagulation or visits to anticoagulation clinics 

15 days before or 90 days after the VTE diagnosis. 
[use described here](https://www.sciencedirect.com/science/article/pii/S095980491200843X) hello to Alex in our team! (@alexwalkerepi )

Filled prescription for anticoagulant within 30 days (unless death from any cause within 30 days)
(From Swedish study above)

Linking anticoagulation but also INR monitoring (ie other evidence of interaction with an anticoagulation clinic). Although arguably less common now as warfarin not first line for acute VTE.
[Link here describing this](https://bpspubs.onlinelibrary.wiley.com/doi/full/10.1046/j.1365-2125.2000.00199.x?sid=nlm%3Apubmed
)

**Next steps**
Describe the coding journey of a typical VTE patient, which are the most prevalent codes
Determine how to distinguish incident VTE, take a look at plots to see if looks sensible using 1 year.

**Comment by speed-vm** [comment]
**Incident vs Prevalent VTE continued**

@acagreen17 and @speed-vm to help with ehrQL for defining incident VTE

**Written definition**
To determine incident from prevalent VTE we:

- excluded those with a VTE diagnosis codes within one year of the first VTE code in the study period.

AND

To ensure a new incident VTE case we:

- only included those with a prescription for an anticoagulant between 15 days before and 90 days after the VTE diagnosis or a date of death within 30 days of the event.

An underlying cause of death of VTE was included as evidence of VTE diagnosis

**Comment by matthewberesford92** [comment]
Looks great @speed-vm . 

Just wondering about the anticoagulation condition. Do you think we will incorrectly exclude some vte diagnosed in secondary care where the prescription hasn’t made it on to the primary care record in time? E.g long admission where patient had vte diagnosed as inpatient/on admission but patient stays in hospital for 90 days. Likely to be small numbers but would be good to check . What do you think?

**Comment by speed-vm** [comment]
@matthewberesford92 Yes, likely low numbers. We can collect this variable in a column and assess with and without
It would be good to go with this as reference'able method (see comment above) and we can always list as a potential limitation

**Comment by matthewberesford92** [comment]
Sounds like a good approach @speed-vm

**Comment by acagreen17** [comment]
Sorry if this has already been discussed but what level are you wanting to report the VTE rate at? Monthly/yearly?

**Comment by matthewberesford92** [comment]
Ideally monthly. We might be able to draw inferences about the impact of the pandemic waves with that granularity but maybe less able if we only have yearly rates

**Comment by acagreen17** [comment]
Some Qs (mostly for @speed-vm ) but @matthewberesford92 or @joshuamutio might also know. Aplogies if these are obvious, I'm still getting up to speed on this project.

1. Why are we including anticoagulant or  _date of death within 30 days of the event_ as an inclusion criteria. This is filtering out patients with a VTE code who did not receive an anticoagulant and died within 30 days of code. Would you not ever expect this to be the case? Surely there are some individuals who die very quickly after a VTE without having received an anticoagulant? I assume we are trying to filter out the codes which correspond to a previous VTE or something? 

2. `An underlying cause of death of VTE was included as evidence of VTE diagnosis` is this an inclusion criteria? Not clear (to me) how this fits in above

3. The incident cases of VTE we're currently identifying are going to be a mixture of true incident cases and recurrent cases of VTE (from before the beginning of the study period), but we're not including any recurrent events during the study period. Is this a problem? Do we want to include recurrent VTEs within the study period or are we happy that this is a very small number (has anyone checked? I know you've checked the number of codes, but what about the spacing between these?)

4. Is there a particular codelist for anticoagulant that you'd suggest using?

**Comment by speed-vm** [comment]
Sorry @acagreen17 - I was not clear. I can answer those questions

1.  Incident VTE events need to have received a prescription for anticoagulation 15days before the event or 90 days after OR date of death within 30 days
As you say, we want to capture those that died from VTE without having received an Rx for anticoagulation. 

2. An underlying cause of death of VTE is part of the main primary clinical outcomes.  When we report incidence just need to make sure deaths from VTE are grouped together with incident cases where patients didn't die.

3. As far as I know no one has checked the intervals between codes looking for recurrence occuring within the study period. In the interests of time, should go for the first case in the study period. 

4. Is there a particular codelist for anticoagulant that you'd suggest using? Yes these two.
https://www.opencodelists.org/codelist/user/matthewberesford92/oral-anticoagulation-used-for-treatment-of-vte-in-primary-care-uk-dmd-dmd/316ca70a/
https://www.opencodelists.org/codelist/user/matthewberesford92/anticoagulation-taken-for-vte-dmd/638b8d74/

**Comment by acagreen17** [comment]
>1. Incident VTE events need to have received a prescription for anticoagulation 15days before the event or 90 days after OR date of death within 30 days
As you say, we want to capture those that died from VTE without having received an Rx for anticoagulation.

But we're not capturing those that **died from VTE** without having received an Rx for anticoagulation, we're capturing those that just died (for any reason) without having received an Rx for anticoagulation. I'm sure this is probably obvious to you guys so apologies for Qs. Just making sure I understand fully.

2. Okay, thanks for clarifying.

3. Okay, worth remembering for limitations when writing up/potential sensitivity analysis/investigation

4. Perfect thanks!

**Comment by speed-vm** [comment]
1. Sorry, long day. I see what you mean. 
So we want to include the first code for a VTE event **IF they died from any cause within the first 30 days**, because in this period they may not have had time to be prescribed an anticoagulant. 

Taking definition [here](https://www.sciencedirect.com/science/article/pii/S095980491200843X )

**Comment by matthewberesford92** [comment]
Thanks for responding @speed-vm . A few thoughts from me @acagreen17 

1. I am interested by the time brackets they used in the EJC paper but it seems reasonable to use their definition as a first pass. We may need to do some form of sensitivity analysis in the fullness of time that plays with the inclusion timings. 

2. Agree, this is key

3. This is the issue that Jon identified. Individuals with VTE codes in the study period can have anywhere from  1 to 30 VTE-related codes . This is most likely to represent duplicate coding for the same incident diagnosis but will in some cases represent a new incident diagnosis. As Jon and I found, very very difficult to differentiate as there is a lack of consistency in the codes used. I settled on the fact that we are going to be comparing the charateristics of two groups of individuals who had _at least one_ incident VTE diagnosis in the study period. 

4. As @speed-vm suggested

**Comment by acagreen17** [comment]
Thanks both. Will implement as described but perhaps we can go through and discuss on Friday to make sure it all makes sense for this study. Just so I can keep track of my thoughts I think it would be good to talk about:

- Including patients with a previous VTE diagnosis (this is different to the study mentioned above which excluded patients with a VTE prior to start date). I don't think this wouldn't bother me so much if we were including multiple VTW events for patients.
- Including a date of death within 30 days of the event as part of the validation criteria. Not sure this works so well when death is the primary outcome (but possibly okay depending on how a VTE-death is defined). This also links in with not including an underlying cause of death of VTE was included as evidence of VTE diagnosis (as per the other study).
-  Only including validated instance of VTE wasn't actually shown to have any significant difference (https://bpspubs.onlinelibrary.wiley.com/doi/epdf/10.1046/j.1365-2125.2000.00199.x?src=getftr). 
)
- Should patients be required to be registered on index date (and have 6/12 months of research standard data available prior)?

**Comment by speed-vm** [comment]
Thank you @acagreen17  Good plan - would be really good to see what you think :) 

- Should patients be required to be registered on index date (and have 6/12 months of research standard data available prior)? - **Yes please**

**Comment by acagreen17** [comment]
Right, study def updated in this branch [here](https://github.com/opensafely/vte-diagnoses-and-deaths/tree/incident-VTE). @joshuamutio shall we go through this together next week (before Fridays meeting) so that you're comfortable developing post Fridays discussions?

**Comment by alexwalkerepi** [comment]
>I am interested by the time brackets they used in the EJC paper

The intention of using evidence of anticoagulation is to increase the specificity of the definition, because cases where treatment is initiated are much less likely to be speculative/suspected coded diagnoses. The fairly wide time margins used are so that the definition is suitably inclusive, and to account for the vagaries of diagnostic recording in EHR data. There's certainly no harm in sensitivity analyses tweaking the definition.

>Only including validated instance of VTE wasn't actually shown to have any significant difference (https://bpspubs.onlinelibrary.wiley.com/doi/epdf/10.1046/j.1365-2125.2000.00199.x?src=getftr).

This paper is sort of 2 papers in one:
1. validating the definition of anticoagulation supported (primary care) VTE diagnoses using manually obtained hospital records;
2. a case control study on VTE incidence on contraceptive users/non-users.

It didn't alter the results for their case control study, but it's entirely possible it could for other questions. It would certainly affect the reported VTE incidence. The main value of it IMO is the validation.

**Comment by matthewberesford92** [comment]
Thanks @alexwalkerepi this is really helpful advice.
And thanks for providing the citation. Old paper but they clearly invested a lot of effort validating their definition!

Interesingly, they dont appear to look at how many events they would have identified (and the specificity of this definition using GP questionaire validation as the gold standard) if they had dropped the anticoag requirement. 

I agree that we should go with the definition used in the EJC and work from there!
